src/chocolatey/infrastructure.app/services/ChocolateyPackageService.cs

```diff
@@ -65,7 +65,7 @@ public class ChocolateyPackageService : IChocolateyPackageService
 @"
 Did you know the proceeds of Pro (and some proceeds from other 
  licensed editions) go into bettering the community infrastructure?
- Your support ensures an active community, it makes you look smarter,
+ Your support ensures an active community, keeping Chocolatey tip top,

```

Comment:
@birgersp I agree that `it makes you look smarter` is a candidate for replacement, but not sure if `keeping Chocolatey tip top` fits the bill.  If nothing else, I think it should be `keeps Chocolatey tip top` in order to better match the context of the sentence.  Thoughts?